include/pros/adi.hpp

```diff
@@ -32,12 +40,27 @@ class ADIPort {
 	 * reached:
 	 * ENXIO - The given value is not within the range of ADI Ports
 	 *
-	 * \param port
+	 * \param adi_port
 	 *        The ADI port number (from 1-8, 'a'-'h', 'A'-'H') to configure
 	 * \param type
 	 *        The configuration type for the port
 	 */
-	ADIPort(std::uint8_t port, adi_port_config_e_t type = E_ADI_TYPE_UNDEFINED);
+	ADIPort(std::uint8_t adi_port, adi_port_config_e_t type = E_ADI_TYPE_UNDEFINED);
+	
+	/**
+	 * Configures an ADI port on an adi expander to act as a given sensor type.
+	 *
+	 * This function uses the following values of errno when an error state is
+	 * reached:
+	 * ENXIO - The given value is not within the range of ADI Ports
+	 *
+	 * \param port
+	 *        The pair of the ADI port number (from 1-8, 'a'-'h', 'A'-'H') and the
+	 *  	  smart port number (from 1-22) to configure 
+	 * \param type
+	 *        The configuration type for the port
+	 */
+	ADIPort(ext_adi_port_pair_t port, adi_port_config_e_t type = E_ADI_TYPE_UNDEFINED);
 
 	virtual ~ADIPort(void) = default;
```

Comment:
Nit: `void` is redundant, including in all the other methods below. I'm not sure what the project style is but `void` parameters exist in some of the headers (motor.hpp) but not others (imu.hpp). Imo it's more readable to omit.

Reply:
To clarify: it's always redundant in the parameter list in C++.

Reply:
i blame @baylessj for this but changing that is out of scope of this PR

Reply:
i take it back, while we're here and refactoring this file this is a good one to just do